src/examples.js

```diff
@@ -42,6 +42,13 @@ export const examples = [
           { cap: 'ែ', key: ['popup', 'popup-default-K_E+shift'] },
           { cap: 'រ', key: 'default-K_R' }
         ],
+        keys_nida_tablet, keys_angkor_tablet: [
+          { cap: 'ខ', key: 'default-K_X' },
+          { cap: '្', key: 'default-K_J' },
+          { cap: 'ម', key: 'default-K_M' },
+          { cap: ['Shift', 'ែ'], key: ['shift', 'shift-K_E'] },
+          { cap: 'រ', key: 'default-K_R' },
+        ],
```

Comment:
This is not valid JavaScript -- you have to copy the definition of keys_angkor_tablet to keys_nida_tablet.

```suggestion
        keys_nida_tablet: [
          { cap: 'ខ', key: 'default-K_X' },
          { cap: '្', key: 'default-K_J' },
          { cap: 'ម', key: 'default-K_M' },
          { cap: ['Shift', 'ែ'], key: ['shift', 'shift-K_E'] },
          { cap: 'រ', key: 'default-K_R' },
        ],
        keys_angkor_tablet: [
          { cap: 'ខ', key: 'default-K_X' },
          { cap: '្', key: 'default-K_J' },
          { cap: 'ម', key: 'default-K_M' },
          { cap: ['Shift', 'ែ'], key: ['shift', 'shift-K_E'] },
          { cap: 'រ', key: 'default-K_R' },
        ],
```

Reply:
I have made the changed.